tests/magma/core/model/py/test_ports.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+import unittest
+from multiprocessing.managers import SharedMemoryManager
+
+import numpy as np
+
+from lava.magma.compiler.channels.interfaces import AbstractCspPort
+from lava.magma.core.model.py.ports import PyInPort, PyInPortVectorDense, \
+    PyOutPortVectorDense, PyOutPort
+
+from lava.magma.compiler.channels.pypychannel import PyPyChannel
+
+
+def get_channel(smm, data, size, name="test_channel") -> PyPyChannel:
+    channel = PyPyChannel(
+        smm=smm, src_name=name, dst_name=name, shape=data.shape,
+        dtype=data.dtype, size=size
+    )
+    return channel
+
+
+class TestPyPorts(unittest.TestCase):
+    def test_port_send_out_vec_dense_to_in_vec_dense(self):
```

Comment:
Change name of test method?

Add tests for 
- [ ] PyInPortVectorSparse
- [ ] PyInPortScalarDense
- [ ] PyInPortScalarSparse